examples/demo-rollup/src/main.rs

```diff
@@ -40,6 +53,14 @@ async fn main() -> Result<(), anyhow::Error> {
 
     // Initialize the Celestia service
     let mut demo_runner = NativeAppRunner::<Risc0Host>::new(rollup_config.runner.clone());
+
+    let storj = demo_runner.inner().current_storage.clone();
```

Comment:
This looks like it depends on an implementation detail of `demo_runner` (that `inner()` returns an AppTemplate which happens to have a field called `current_storage`. Ideally, all of this should be implemented purely in terms of trait methods. Can we add a trait method to `StateTransitionRunner` which returns a handle to storage?